sdk/identity/identity-broker/samples/v1/typescript/package.json

```diff
@@ -37,7 +37,7 @@
     "@types/node": "^18.0.0",
     "@eslint/js": "^9.9.0",
     "eslint": "^9.9.0",
-    "typescript-eslint": "~8.23.0",
+    "typescript-eslint": "~8.24.0",
```

Comment:
Why is ESLint in a sample here?  Doesn't quite make sense for us to have them in a sample for any ESLint related items

Reply:
I don't know the reason, I have been just keeping them updated.  Seems to be introduced by PR #26091

Reply:
could be a mistake, or old bug in dev-tool.  I will remove the linting for this sample.

Reply:
Removed